standard-covoiturage_openapi.yaml

```diff
@@ -116,6 +128,301 @@ paths:
+      operationId: getPassengerRegularTrips
+      parameters:
+        - $ref: '#/components/parameters/departureLat'
+        - $ref: '#/components/parameters/departureLng'
+        - $ref: '#/components/parameters/arrivalLat'
+        - $ref: '#/components/parameters/arrivalLng'
+        - $ref: '#/components/parameters/departureTimeOfDay'
+        - $ref: '#/components/parameters/departureWeekdays'
+        - $ref: '#/components/parameters/timeDelta'
+        - $ref: '#/components/parameters/departureRadius'
+        - $ref: '#/components/parameters/arrivalRadius'
+        - $ref: '#/components/parameters/minDepartureDate'
+        - $ref: '#/components/parameters/maxDepartureDate'
+        - $ref: '#/components/parameters/count'
+      responses:
+        200:
+          description: Ok. Request processed successfully.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/PassengerTrip'
+                    - type: object
+                      properties:
+                        schedules:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/Schedule'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        429:
+          $ref: '#/components/responses/TooManyRequests'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /messages:
+    post:
+      tags:
+      - Interact
+      summary: Send a message to the owner of a retrieved journey.
+      description: Route used to allow a user to connect back to the owner of a retrieved journey through a texte message.
+      operationId: postConnections
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/ConnectionMessage'
+              type: object
+              required:
+                - message
+                - recipientCarpoolerType
+              properties:
+                recipientCarpoolerType:
+                  type: string
+                  enum:
+                    - driver
+                    - passenger
+                  default: driver
+                  description: Defines if the recipient of this message is either the driver or the passenger.
+      responses:
+        201:
+          description: Successful operation.
+        404:
+          description: The targeted journey or user no more exists.
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        429:
+          $ref: '#/components/responses/TooManyRequests'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+
+  /bookings:
+    post:
```

Comment:
If both send a `Booking` object I think that we could merge POST and PUT operations into a single PUT operation. However, I am rather in favor of `POST`ing a `BookingRequest` and eventually `PUT`ting `Booking` updates. 

Excerpt from [my comment on the other PR](https://github.com/fabmob/standard-covoiturage/pull/5#discussion_r870541576): 

----
I read [here](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html) a definition that sounds quite right for upsert :

 >  The PUT method requests that the enclosed entity be stored under the supplied Request-URI. If the Request-URI refers to an already existing resource, the enclosed entity SHOULD be considered as a modified version of the one residing on the origin server. If the Request-URI does not point to an existing resource, and that URI is capable of being defined as a new resource by the requesting user agent, the origin server can create the resource with that URI.

Reply:
We decided to drop the PUT operation so there is no more subject here.